src/main/java/org/example/models/Patient.java

```diff
@@ -0,0 +1,89 @@
+package org.example.models;
+import jakarta.persistence.*;
```

Comment:
Remove wildcard imports (there is a guide how to disable this feature in Idea, in training team channel)

Reply:
https://t.me/c/1529326329/134

Reply:
Done